### PR TITLE
add int to date feature

### PR DESCRIPTION
**Pull request by coseyo**
For some developers will use mysql int type to store the time, this feature could convert the mysql int to es date type.

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @coseyo 

Please add a test.

File: river/sync.go [hunk 1]
```diff
@@ -479,3 +462,33 @@ func (r *River) doBulk(reqs []*elastic.BulkRequest) error {
...
+		} else {
+			fieldValue = v
+		}
+
+	case fieldTypeDate:
+		if col.Type == schema.TYPE_NUMBER {
+			col.Type = schema.TYPE_DATETIME
+			switch value := value.(type) {
+			case int64:
+				fieldValue = r.makeReqColumnData(col, time.Unix(value, 0).Format("2006-01-02 15:04:05"))
```
Comment (siddontang) [inline river/sync.go]: use mysql.TimeFormat direclty

File: river/sync.go [hunk 2]
```diff
@@ -479,3 +462,33 @@ func (r *River) doBulk(reqs []*elastic.BulkRequest) error {
...
+			fieldValue = strings.Split(str, ",")
+		} else {
+			fieldValue = v
+		}
+
+	case fieldTypeDate:
+		if col.Type == schema.TYPE_NUMBER {
+			col.Type = schema.TYPE_DATETIME
+			switch value := value.(type) {
+			case int64:
```
Comment (siddontang) [inline river/sync.go]: why we only care int64 and int32 here?

Comment (coseyo) [reply river/sync.go]: oh, change it to
```
if col.Type == schema.TYPE_NUMBER {
			col.Type = schema.TYPE_DATETIME
			switch value := value.(type) {
			case int64:
				fieldValue = r.makeReqColumnData(col, time.Unix(value, 0).Format("2006-01-02 15:04:05"))
			case int:
				fieldValue = r.makeReqColumnData(col, time.Unix(int64(value), 0).Format("2006-01-02 15:04:05"))
			case int8:
				fieldValue = r.makeReqColumnData(col, time.Unix(int64(value), 0).Format("2006-01-02 15:04:05"))
			case int16:
				fieldValue = r.makeReqColumnData(col, time.Unix(int64(value), 0).Format("2006-01-02 15:04:05"))
			case int32:
				fieldValue = r.makeReqColumnData(col, time.Unix(int64(value), 0).Format("2006-01-02 15:04:05"))
			}
		}
```

File: river/sync.go (hunk 1 above)
Comment (coseyo) [reply river/sync.go]: What's the mysql.TimeFormat mean?

Comment (siddontang) [reply river/sync.go]: `mysql.TimeFormat` is the const name for "2006-01-02 15:04:05"

File: river/sync.go (hunk 2 above)
Comment (siddontang) [reply river/sync.go]: seem you can extract to a function like `toInt64(v) (int64, bool)` so we can simplify the switch case.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**